thruster-server/src/hyper_server.rs

```diff
@@ -55,6 +55,7 @@ impl<T: Context<Response = Response<Body>> + Send> ThrusterServer for HyperServe
             server.await?;
 
             Ok::<_, hyper::Error>(())
-        });
+        })
+        .unwrap();
```

Comment:
Let's make this an expect instead of an unwrap, that way we can control the error a little more succinctly.